app.js

```diff
@@ -7,6 +7,29 @@ const bodyParser = require('body-parser');
 const session = require('express-session');
 const flash = require('connect-flash');
 const url = require('url');
+const nodemailer = require('nodemailer');
+
+
+function confirmation(email) {
```

Comment:
This should have two arguments. One is the email and one should be a confirmation string that will be passed from the server side.